helm.yaml

```diff
@@ -0,0 +1,32 @@
+name: helm
+description: Kubernetes package manager
+homepage: https://helm.sh/
+repository: ''
```

Comment:
I think you can use https://github.com/helm/helm/ here. This helps to automatically update the package.